deploy/install.sh

```diff
@@ -312,11 +315,19 @@ apiRequest() {
   url=$2
   json=$3
 
-  response="$(curl -sS -X "${method}" "${API_URL}${url}" \
-    -H "accept: application/json; charset=utf-8" \
-    -H "Authorization: Api-Token ${API_TOKEN}" \
-    -H "Content-Type: application/json; charset=utf-8" \
-    -d "${json}")"
+  if "$SKIP_CERT_CHECK" = "true"; then
+    response="$(curl -k -sS -X ${method} "${API_URL}${url}" \
+      -H "accept: application/json; charset=utf-8" \
+      -H "Authorization: Api-Token ${API_TOKEN}" \
+      -H "Content-Type: application/json; charset=utf-8" \
+      -d "${json}")"
+  else
+    response="$(curl -sS -X ${method} "${API_URL}${url}" \
+      -H "accept: application/json; charset=utf-8" \
+      -H "Authorization: Api-Token ${API_TOKEN}" \
+      -H "Content-Type: application/json; charset=utf-8" \
+      -d "${json}")"
+  fi
```

Comment:
The if-else block can be avoided by introducing a curl options variable. This would improve readability and maintainability a lot.